libenv/unix_iface.c

```diff
@@ -711,13 +711,15 @@ static void FindV6InterfacesInfo(EvalContext *ctx, Rlist **interfaces, Rlist **h
                 // StringCopy effectively returns the length of the src string,
                 // up to (inclusive) a maximum of passed in buffer size
 
+#ifndef NDEBUG
                 // We know there was more data, at least a colon:
                 assert(src_length == bytes_to_copy);
                 const size_t dst_length = src_length - 1;
 
                 // We copied everything up to, but not including, the colon:
                 assert(ifconfig_line[dst_length] == ':');
                 assert(current_interface[dst_length] == '\0');
+#endif // not NDEBUG
```

Comment:
Use this macro instead
```suggestion
                // We know there was more data, at least a colon:
                assert(src_length == bytes_to_copy);
                NDEBUG_UNUSED const size_t dst_length = src_length - 1;

                // We copied everything up to, but not including, the colon:
                assert(ifconfig_line[dst_length] == ':');
                assert(current_interface[dst_length] == '\0');
```

Reply:
That looks great but how would I have found that? ;) Hmm...